.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,11 @@
 - **Etcher version:** 
 - **Operating system and architecture:** 
 - **Image flashed:** 
+- **What do you think should have happened:** <!-- even step by step reproduction process -->
+- **What happened:**
 - **Do you see any meaningful error information in the DevTools?** 
-
 <!-- You can open DevTools by pressing `Ctrl+Shift+I` (`Ctrl+Alt+I` for Etcher before v1.3.x), or `Cmd+Opt+I` if you're on macOS. -->
+
+<!-- issues with missing information will be labeled as not-enough-info and closed shortly -->
+<!-- plase try to include as many influencing elements as possible are you root, does any other process block the device, etc. -->
```

Comment:
"plase" -> "please"? Should probably be a colon after "possible"? And "factors" might be a better word than "elements"?